test/e2e-go/cli/goal/expect/statefulTealCreateAppTest.exp

```diff
@@ -114,6 +114,17 @@ proc statefulTealTest { TEST_ALGO_DIR TEST_DATA_DIR TEAL_PROGRAM} {
     # Delete application should still succeed
     ::AlgorandGoal::AppDelete $APP_ID $WALLET_1_NAME $WALLET_1_PASSWORD $ACCOUNT_1_ADDRESS "str:hello" $TEST_PRIMARY_NODE_DIR
 
+    # App create with extra pages option should succeed
+    ::AlgorandGoal::AppCreateExPages $PRIMARY_WALLET_NAME "" $PRIMARY_ACCOUNT_ADDRESS ${TEAL_PROGS_DIR}/${TEAL_PROGRAM} "str:hello" $GLOBAL_BYTE_SLICES $LOCAL_BYTE_SLICES ${TEAL_PROGS_DIR}/clear_program_state.teal $TEST_PRIMARY_NODE_DIR 2
```

Comment:
Again, we don't want to just test that extra pages value is set, but that because it's set specifying a large program is allowed. So you need to feed it a big approval program.  (Which may require changing goal to allow such a long program when the proto allows extra-pages)

Reply:
what's the simplest way to generate a large teal file? or is there one I could use?

Reply:
Since this command is expecting teal source code, you can create a file that starts "#pragma version 3" and then has as many lines of "int 1" as you'd like. Each "int 1" will contribute one more byte to the program length.

This app won't return success is executed (because the stack may overflow, and if it doesn't it still won't end properly) but that won't matter for your tests.

Reply:
I have some code in tealConsensusTest.exp that creates long teal programs.